.github/workflows/scala.yml

```
@@ -0,0 +1,36 @@
+          java-version: '21'
+          distribution: 'temurin'
+          cache: 'sbt'
+      - name: Install sbt
+        run: |
+          echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | sudo tee /etc/apt/sources.list.d/sbt.list
+          echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | sudo tee /etc/apt/sources.list.d/sbt_old.list
+          curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x99E82A75642AC823" | sudo apt-key add
+          sudo apt-get update
+          sudo apt-get install sbt
+      - name: Run tests
+        working-directory: ./scala/coding  
```

yamllint

[error] trailing spaces

(trailing-spaces)